ui/apps/platform/src/routePaths.ts

```diff
@@ -41,6 +42,7 @@ export const configManagementPath = `${mainPath}/configmanagement`;
 export const dashboardPath = `${mainPath}/dashboard`;
 export const dataRetentionPath = `${mainPath}/retention`;
 export const exceptionConfigurationPath = `${mainPath}/exception-configuration`;
+export const exceptionManagementPath = `${vulnerabilitiesBasePath}/exception-management`;
```

Comment:
* Plus: path segment is exception-management consistent with recently added exception-configuration
* Minus: title **Deferral Management** is inconsistent with both its own route and its configuration. Would **Exception Management** be more consistent and also remove ambiguity of **Deferral** referring to both deferral and false positive requests?

Reply:
Yes, I think we have decided to go with Exception Management so it makes more sense in terms of including both deferral and false positive requests